voucher/src/main/res/layout/full_voucher_view.xml

```diff
@@ -91,51 +93,55 @@
         app:layout_constraintTop_toBottomOf="@id/paymentReferenceSeparator"
         tools:text="12345.1234.12345.12345.12345678901 123456789012345" />
 
+    <Space
+        android:id="@+id/space_buttons"
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/standard_margin"
+        app:layout_constraintTop_toBottomOf="@id/textView_reference_code" />
+
+    <androidx.constraintlayout.helper.widget.Flow
+        android:id="@+id/layout_buttons"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/standard_half_margin"
+        android:layout_marginEnd="@dimen/standard_half_margin"
+        app:constraint_referenced_ids="button_copyCode,button_store"
+        app:flow_horizontalGap="@dimen/standard_half_margin"
+        app:flow_horizontalStyle="spread_inside"
+        app:flow_wrapMode="chain"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/space_buttons" />
+
     <com.google.android.material.button.MaterialButton
         android:id="@+id/button_copyCode"
-        style="@style/AdyenCheckout.Voucher.ButtonCopyCode"
+        style="@style/AdyenCheckout.Voucher.ActionButton"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        app:layout_constraintEnd_toStartOf="@+id/button_downloadPdf"
-        app:layout_constraintHorizontal_chainStyle="packed"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@id/textView_reference_code" />
+        android:text="@string/checkout_voucher_copy_code" />
```

Comment:
Generally, with components we don't set the text attrbute in the layout xml files, we set them in `styles.xml` instead. That is to allow merchants to override the resource there if needed.
So we use `tools:text` instead of `android:text` in this file.
One other advantage of this (more like a side effect) is that we can spot localization issues like the one I mentioned in the previous comment, because the text will simply not appear if you forgot to localize it and you used `tools:text`. I can also explain more later.